internal/endtoend/testdata/pg_timezone_names/query.sql

```diff
@@ -0,0 +1,8 @@
+-- name: GetTimezones :many
+SELECT * from pg_catalog.pg_timezone_names;
+
+-- name: GetTables :many
+SELECT table_name::text from information_schema.tables;
```

Comment:
I had to cast these to `::text` since I think the information_schema uses a database type that is not currently recognized by sqlc (it used `interface{}`).  It might be a simple fix to change that.